cscs-checks/libraries/math/trilinos_compile_run.py

```diff
@@ -49,3 +51,13 @@ def __init__(self, linkage):
     def set_cxxflags(self):
         flags = self.prgenv_flags[self.current_environ.name]
         self.build_system.cxxflags = flags
+
+    @rfm.run_before('compile')
+    def set_prebuild_cmd(self):
+        if self.current_environ.name.startswith('PrgEnv-intel'):
+            if '20.06' in os.getenv('MODULERCFILE', ''):
+                self.modules += ['cray-netcdf-hdf5parallel']
```

Comment:
Why this is needed?

Reply:
The module load is needed in order to have the CRAY_NETCDF_HDF5PARALLEL_PREFIX variable set.